apps/nextjs-website/react-components/components/Accordion/AccordionItem.tsx

```diff
@@ -23,6 +23,13 @@ export const AccordionItem: React.FC<AccordionItemProps> = ({
       ? palette.primary.main
       : palette.custom.primaryColorDark;
 
+  const appendItemIDToURLHash = () => {
+    if (!itemID) return;
+    const url = new URL(window.location.href);
+    url.hash = itemID;
+    window.history.replaceState(null, '', url.toString());
```

Comment:
```suggestion
    window.location.hash = itemID;
```
Solves the following error when clicking an Accordion with a URL hash already present: `TypeError: Cannot read properties of null (reading '__PRIVATE_NEXTJS_INTERNALS_TREE')`